src/rules/alias.test.ts

```diff
@@ -57,6 +57,13 @@ tester.run('paths-alias', rule, {
 			errors: ['Update import to @foo/x/y/z'],
 			output: `import z from '@foo/x/y/z';`,
 		},
+		{
+			name: 'relative named import from alias must be fixed',
+			filename: path.resolve('./src/nested/index.ts'),
+			code: `import { x } from '../qux/x/y/z';`,
+			errors: ['Update import to @qux/x/y/z'],
+			output: `import { x } from '@qux/x/y/z';`,
+		},
```

Comment:
What difference of test that described below?

```
		{
			name: 'relative import from alias used in another alias must be fixed',
			filename: path.resolve('./src/foo/index.ts'),
			code: `import z from '../bar/x/y/z';`,
			errors: ['Update import to @bar/x/y/z'],
			output: `import z from '@bar/x/y/z';`,
		},
```

Could you elaborate for what case this test for

Reply:
Yea so this one was trickier to explain but it's really testing the `tsconfig` alias path more than the import itself. It is testing the change below.

Reply:
I can verify that this test is useful and can catch a problem if revert fix below

![image](https://github.com/user-attachments/assets/e89fddae-f702-4a32-8e71-43cd4375095d)